CONTRIBUTING.md

```diff
@@ -2,21 +2,53 @@
 
 Observable Plot is released under the ISC license. You are welcome to fork this repository and to send us pull requests to contribute bug fixes or new features.
 
-To develop Observable Plot locally, clone this repository and install its dependencies:
+## Set up
+To set up your local development environment for Plot, follow these steps on your terminal:
 
-```
+```bash
+# 1. Clone this repository 
 git clone git@github.com:observablehq/plot.git
-cd plot
-yarn
+cd plot # cd into the folder
+
+# 2. Checkout a new development branch, by convention `user-name/feature-name`
+git checkout -b user-name/feature-name
+
+# 3. Install dependencies listed in the package.json file
+yarn 
 ```
 
+## Development
 Plot is written in ES modules and uses [Vite](https://vitejs.dev/) for development; this means that you can edit the Plot source code and examples, and they’ll update live as you save changes. To start Vite:
 
 ```
 yarn dev
 ```
 
-This should open http://localhost:8008/ where you can browse the tests.
+This should open http://localhost:8008/ in your browser where you can see the tests plots defined in `test/plots`. 
+
+![local development environment](img/localhost.png)
+
+If you'd like to create a new test plot as part of your development, you need to:
+
+1. Create a new `.js` file in `test/plots`, e.g., `test/plots/new-example.js` (an easy start is duplicating an existing example).
+2. Export the function from your `new-example.js` file in `test/index.js`. Because the first alphabetical option is displayed by on the local server, you may want to export the example as `_newExample`.
```

Comment:
I would not recommend this, as then folks are likely to submit a pull request with this naming convention. We want the tests to be in alphabetical order in `test/plots/index.js`. Also, once you’ve selected your test, you get hot module reloading, so you don’t have to keep re-selecting your test as you develop. And, you can use the left and right arrow keys to flip through the tests quickly, as well as Enter + typing to quickly select your desired test via the keyboard.

Reply:
The test file is `test/plots/index.js`, not `test/index.js`.

Reply:
Noted about the naming convention (I hadn't used the hot-reloading enough to realize it kept your Select option). I'll remove that content.

Reply:
And I'll update the screenshot :)